source/_patterns/10-atoms/navigation/expand-icon.scss

```diff
@@ -1,25 +1,8 @@
-.expand-icon__icon {
-  width: 17px;
-  height: 9px;
-  display: block;
-  background: transparent url('../img/arrows/menu_arrow_white.svg') no-repeat 0 0 / 17px;
-  margin-left: 8px;
-  transition: transform 0.4s ease;
-}
+.expand-icon {
+  display: flex;
 
-.expand-icon--open {
   .expand-icon__icon {
-    transform: translateZ(0) rotate(180deg); // HACK: «translateZ(0)» is because of 1px-jumping-bug in Safari (rechstst)
-  }
-
-  .expand-icon__text--to-open,
-  .expand-icon__a11y-open {
-    display: none;
-  }
-
-  .expand-icon__text--to-close,
-  .expand-icon__a11y-close {
-    display: block;
+    @include caret('white', 'down', 'medium');
```

Comment:
der Safari macht jetzt keine Hüpfer mehr, oder? (wegen dem gelöschten Kommentar)

Reply:
Nein, sehe nix mehr im Safari: https://frontend-framework-stag-pr-392.herokuapp.com/?p=pages-landingpage

Reply:
meiner macht auch keinen Hüpfer mehr (ich glaub es war in der Navi)